src/Uintra20/Content/uintra20/src/app/feature/project/specific/groups/group-details-wrapper/group-details-wrapper.component.html

```diff
@@ -0,0 +1,34 @@
+<div class="groups">
+  <div *ngIf="data" class="groups-header-wrapper">
+    <div class="groups-header">
+      <h2 class="groups-header__title">
+        <a
+          [routerLink]="data.title.link.originalUrl | ulink"
+          class="groups-header__title-link">
+          {{data.title.title}}
+        </a>
+      </h2>
+      <ul *ngIf="data.groupLinks" class="tabs col-md-8">
+        <li *ngFor="let tab of data.groupLinks" class="tabs__item">
+          <a
+            [routerLink]="tab.link.originalUrl | ulink"
+            class="tabs__item-link"
+            routerLinkActive="active"
+            [routerLinkActiveOptions]="{ exact: true }">
```

Comment:
It makes sense? Subpages should be have an active class in the current tab

Reply:
+